From: Danny Al-Gaaf <danny.al-gaaf@bisect.de>
Date: Mon, 22 Feb 2016 15:17:20 +0100
Subject: librbd/exclusive_lock/AcquireRequest.cc: init lock_type

Fixup for:

CID 1351687 (#1 of 1): Uninitialized scalar variable (UNINIT)
 var_decl: Declaring variable lock_type without initializer
 uninit_use: Using uninitialized value lock_type.

Signed-off-by: Danny Al-Gaaf <danny.al-gaaf@bisect.de>
(cherry picked from commit da9ede90cd257ff605ab3ebfcc1aa995f655f1cc)
(cherry picked from commit c868d4cf2afea9b948b780d76a5b2ce606139a8f)

Resolves: rhbz#1408226
---
 src/librbd/exclusive_lock/AcquireRequest.cc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/librbd/exclusive_lock/AcquireRequest.cc b/src/librbd/exclusive_lock/AcquireRequest.cc
index 5b5ba67..1887e74 100644
--- a/src/librbd/exclusive_lock/AcquireRequest.cc
+++ b/src/librbd/exclusive_lock/AcquireRequest.cc
@@ -417,7 +417,7 @@ Context *AcquireRequest<I>::handle_get_lockers(int *ret_val) {
 
   std::map<rados::cls::lock::locker_id_t,
            rados::cls::lock::locker_info_t> lockers;
-  ClsLockType lock_type;
+  ClsLockType lock_type = LOCK_NONE;
   std::string lock_tag;
   if (*ret_val == 0) {
     bufferlist::iterator it = m_out_bl.begin();
